cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt

```diff
@@ -0,0 +1,513 @@
+        val id = resources.getIdentifier("config_showNavigationBar", "bool", "android")
+        return id > 0 && resources.getBoolean(id)
+    }
+
+    private fun setNavigationCameraControlsPadding() {
+        ofNullable(mCameraFragmentBinding.cameraFragmentBottomLinearLayout).executeIfPresent { navigationCamera ->
+            navigationCamera.setPadding(
+                    navigationCamera.paddingLeft,
+                    navigationCamera.paddingTop,
+                    navigationCamera.paddingRight,
+                    navigationCamera.bottom + convertDpToPixels(48))
+        }
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        mSaveOnlySnackbar = Snackbar.make(mCameraFragmentBinding.root,
+                getString(R.string.camera_snackbar_save_only_mode_message),
+                Snackbar.LENGTH_SHORT)
+        mStantSaveModeSnackbar = Snackbar.make(mCameraFragmentBinding.root,
+                getString(R.string.camera_snackbar_stant_save_mode_message),
+                Snackbar.LENGTH_SHORT)
+
+        setButtonsClick(mCameraFragmentBinding.cameraFragmentCloseImageView,
+                mCameraFragmentBinding.cameraFragmentChangeSavePicturesMode,
+                mCameraFragmentBinding.cameraFragmentCaptureImageButton,
+                mCameraFragmentBinding.cameraFragmentGalleryImageView,
+                mCameraFragmentBinding.cameraFragmentSaveImageView)
+
+        setAdapter(mCameraFragmentBinding.cameraPhotosRecyclerView)
+
+        setCameraSetup(mCameraFragmentBinding.cameraFragmentSwitchFlashImageView,
+                mCameraFragmentBinding.cameraFragmentZoomSeekBar,
+                mCameraFragmentBinding.cameraFragmentSwitchLensImageView)
+
+        configureOrientationListener(mCameraFragmentBinding.cameraFragmentCloseImageView,
+                mCameraFragmentBinding.cameraFragmentChangeSavePicturesMode,
+                mCameraFragmentBinding.cameraFragmentCaptureImageButton,
+                mCameraFragmentBinding.cameraFragmentGalleryImageView,
+                mCameraFragmentBinding.cameraFragmentSaveImageView,
+                mCameraFragmentBinding.cameraFragmentSwitchLensImageView,
+                mCameraFragmentBinding.cameraFragmentChipLinearLayout,
+                mCameraFragmentBinding.cameraFragmentSwitchFlashImageView)
+
+        updateCounters()
+    }
+
+    private fun setButtonsClick(closeButton: ImageView, changeSavePicturesReason: ImageView,
+                                captureButton: ImageButton, openGalleryButton: ImageView, savePhotosButton: ImageView) {
+        closeButton.setOnClickListener { closeCamera() }
+
+        changeSavePicturesReason.setOnClickListener{
+            mSaveMode = if (mSaveMode.mode == STANT_MODE) {
+                changeSavePicturesReason.setImageDrawable(ContextCompat.getDrawable(context!!, R.drawable.ic_stant_disabled))
```

Comment:
this icons can be provided by those use the library, add dinamical variable to icon

Reply:
Fixed